src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,43 @@
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+  }
+
+  &__link {
+    position: relative;
+    display: flex;
+    align-items: center;
+    padding: 23px 0;
+    text-transform: uppercase;
+    text-decoration: none;
+    font-weight: 500;
+    color: $color-dark;
+
+    &:hover {
+      color: $color-blue;
+    }
+
+    &.is-active {
```

Comment:
let's add --active modifier and move these styles for those modifier
and will use is-active class only in html for tests passing

Reply:
ok